api/BUILD

```diff
@@ -1,3 +1,17 @@
+# Copyright 2019, OpenTelemetry Authors
```

Comment:
Someone please remind me: do we need the license comment on basically every file?